src/rez/pip.py

```diff
@@ -242,8 +254,22 @@ def pip_install_package(source_name, pip_version=None, python_version=None,
     _cmd(context=context, command=cmd)
     _system = System()
 
+    def pure_python_package(installed_dist):
+
+        true_table = {
+            "true": True,
+            "false": False
+        }
+
+        packages = pkg_resources.find_distributions(destpath)
+        dist = next((package for package in packages if package.key == installed_dist.key), None)
+        wheel_data = dist.get_metadata('WHEEL')
+        wheel_data = Parser().parsestr(wheel_data)
+
+        return true_table[wheel_data["Root-Is-Purelib"]]
```

Comment:
Maybe add a little link to https://www.python.org/dev/peps/pep-0427/#what-s-the-deal-with-purelib-vs-platlib or something else that explains a little bit more what's the deal with `Root-Is-Purelib`?

Reply:
Makes sense I will add it.